exonum-java-binding/core/rust/ejb-app/src/node_builder.rs

```diff
@@ -41,30 +40,18 @@ fn service_factories() -> HashMap<String, Box<ServiceFactory>> {
         TIME_SERVICE.to_owned(),
         Box::new(TimeServiceFactory) as Box<ServiceFactory>,
     );
-    service_factories.insert(
-        EJB_SERVICE.to_owned(),
-        Box::new(JavaServiceFactory) as Box<ServiceFactory>,
-    );
     service_factories
 }
 
-#[doc(hidden)]
-pub fn services_to_enable<P: AsRef<Path>>(path: P) -> HashSet<String> {
-    // Return default list if config file not found.
-    let mut services = load_enabled_services(path).unwrap_or_else(|_| {
-        let mut services = HashSet::new();
-        services.insert(CONFIGURATION_SERVICE.to_owned());
-        services
-    });
-
-    // Add EJB_SERVICE if it's missing
-    services.insert(EJB_SERVICE.to_owned());
+pub fn create() -> fabric::NodeBuilder {
+    let EjbAppServices {
+        enabled_services,
+        user_services,
+    } = load_services_definition(PATH_TO_SERVICES_DEFINITION)
+        .expect("Unable to load services definition");
```

Comment:
Hmm... It can fail not only because of missing file. I'd probably left it as is since `.expect()` includes original cause and in this case we'll get smth like `Unable to load services definition` (general error) and then detailed description of error: missing file, non-UTF symbols, wrong format whatever. WDYT?